portal-ui/src/screens/Console/Buckets/BucketDetails/SetAccessPolicy.tsx

```diff
@@ -30,6 +30,7 @@ import { ErrorResponseHandler } from "../../../../common/types";
 import api from "../../../../common/api";
 import ModalWrapper from "../../Common/ModalWrapper/ModalWrapper";
 import SelectWrapper from "../../Common/FormComponents/SelectWrapper/SelectWrapper";
+import CodeMirrorWrapper from "../../Common/FormComponents/CodeMirrorWrapper/CodeMirrorWrapper";
 
 const styles = (theme: Theme) =>
```

Comment:
```suggestion
const styles = (theme: Theme) =>
  createStyles({
    codeMirrorContainer: {
      marginBottom: 20,
      paddingLeft: 15,
      "&:nth-child(2) .MuiGrid-root:nth-child(3)": {
        border: "1px solid #EAEAEA",
      },
      "& label": {
        marginBottom: ".5rem",
      },
      "& label + div": {
        display: "none",
      },
    },
    ...modalStyleUtils,
    ...spacingUtils,
  });
```
`codeMirrorContainer`  style needs to be defined. to get a border around it.